elements/lisk-chain/src/chain.ts

```diff
@@ -358,12 +358,12 @@ export class Chain {
 		this.dataAccess.resetBlockHeaderCache();
 	}
 
-	public async newStateStore(): Promise<StateStore> {
+	public async newStateStore(heightOffset: number = 0): Promise<StateStore> {
```

Comment:
Please make it named argument. Its super confusing to read the usage without it. 

```
await this.chainModule.newStateStore(1)
```
vs 
```
await this.chainModule.newStateStore({ heightOffset : 1})
```